rules/windows/builtin/firewall_as/win_firewall_as_add_rule_susp_folder.yml

```diff
@@ -0,0 +1,27 @@
+title: New Firewall Exception Rule Added For A Suspicious Folder
+id: 9e2575e7-2cb9-4da1-adc8-ed94221dca5e
+related:
+    - id: cde0a575-7d3d-4a49-9817-b8004a7bf105
+      type: derived
+status: experimental
+description: Detects the addition of a rule to the Windows Firewall exception list where the application resides in a suspicious folder
+references:
+    - https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-r2-and-2008/dd364427(v=ws.10)
+    - https://app.any.run/tasks/7123e948-c91e-49e0-a813-00e8d72ab393/#
+author: frack113
+date: 2023/02/26
+logsource:
+    product: windows
+    service: firewall-as
+detection:
+    selection:
+        EventID: 2004
+        ApplicationPath|contains:
+            - '\AppData\'
+            - '\temp\'
+    filter_block:
+        Action: 2
```

Comment:
A couple of things
- AppData contains legit applications and they add exceptions all the time. Examples: `teams.exe`, `jetbrains-toolbox.exe`, `code.exe`...etc. So this can't be high. It's medium at best without a filter list.
- While executing the command you put, I didn't get EID 2004 but I got EID 2071 - `A rule has been added to the Windows Defender Firewall exception list.`

Reply:
1/ From ETW manifest: 
- 2004 windows 10
- 2071 windows 11

I have test only on a windows 10 VM.

2/ AppData
I don't see an easy way for now, can start as medium.

Reply:
If we put it to medium then we have cde0a575-7d3d-4a49-9817-b8004a7bf105 that covers the same logic.

Reply:
Let's think of a way to keep it and avoid 100% overlapping coverage. Since this is high if we filter the stuff.